test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -77,6 +80,8 @@ type testCase struct {
 	// TODO(#93792): reduce config toil by having a default pod and node spec per
 	// testCase? CreatePods and CreateNodes ops will inherit these unless
 	// manually overridden.
+	// SchedulerConf is the path of scheduler configuration
+	SchedulerConf string
```

Comment:
`SchedulerConfig` or `SchedulerConfigFile` to make it consistent with the other field :)

Reply:
Done